src/app2/webpack.config.js

```diff
@@ -1,119 +0,0 @@
-const fs = require('fs');
```

Comment:
I just went back to one webpack config for everything because it's a bit simpler. I do see many advantages to multiple webpack configs (one for each child application), and in fact that's how we do it at Canopy in production. However, it's a bit trickier to set up so I just went back to a simplistic approach. Let me know if you want to do multiple webpack configs though and we can talk.